requirements/specification.md

```diff
@@ -174,7 +181,9 @@ The fields of _DataSetMetaData_ should be mapped as follows:
 Each DataSet need an Field called "virual_id" which contains the BrowsePath.
 (e.g.; `5:Production.5:ActiveProgram.5:State`)
 
-If two nodes have the same _BrowsePath_ an iterator (".Number") can be send to avoid collisions (e.g., `Parent/Tool.1`, `Parent.3/Tool.2` `Parent3/Tool.3` )
+The `NamespaceIndex` of the `QulifiedName` are mapped to the Namesapces array of the `DataTypeSchemaHeader`
```

Comment:
@ccvca @wlkrm What do you think should we use the NamespaceIndex as described or the nsu and if the namespaceindex -> should possibly each dataset should have an other namespace array?

Reply:
When the index does not change and represent the OPC UA-Server NS-Index, using indices instead of uri is fine.